EventListener/SerializerListener.php

```diff
@@ -26,7 +26,7 @@ public function onKernelRequest(GetResponseEvent $event)
         try {
             switch ($request->attributes->get('xapi_serializer')) {
                 case 'statement':
-                    $request->attributes->set('statement', $this->serializer->deserialize($request->getContent(), 'Xabbuh\XApi\Model\Statement', 'json'));
+                    $request->attributes->set('statement', $this->serializerRegistry->getStatementSerializer()->deserializeStatement($request->getContent()));
                     break;
             }
         } catch (BaseSerializerException $e) {
```

Comment:
While seeing this I think it is time to introduce some custom exceptions in the `php-xapi/serializer` package and catch these here instead of catching implementation specific exceptions here.